src/App/App.tsx

```diff
@@ -87,56 +105,93 @@ class App extends Component<Props, State> {
     return '--/--/----';
   }
 
+  handleComparisonClick = (dropdownValue: ComparisonCategory): void => {
+    let comparisonStats = comparisonData.find(datum => {
+      return datum.category === dropdownValue;
+    })
+    this.setState({ selectedComparison: comparisonStats })
+  }
+
+  blockUnintededRoutes = (): string => {
```

Comment:
We should rename this to something like 'generatePossibleEndpoints' - it would be a more semantic title.